### ISSUE TITLE
Dupe with Loot Pet

### ISSUE THREAD

**Issue by cjm721** [capnkirok/Inventory-Pets#178]
Version 1.11.2-1.4.9.5

If you drop an item and let a mob pick it up (zombie) and then kill the mob you will get two of the item you gave it.

**Comment by capnkirok** [comment]
Yeah, could see this as a problem, though it's working as intended. Have to see if there's anything I can do about it.

**Comment by cjm721** [comment]
How are you getting the drops? Could get the loot table for the entity and then only double items that show up in the loot table

**Comment by capnkirok** [comment]
I take it from the drops event, which pulls everything that was dropped (and dupes it).

**Comment by cjm721** [comment]
EntityLiving::getLootTable then 
```LootTable loottable = this.world.getLootTableManager().getLootTableFromLocation(resourcelocation);```

Can look at EntityLiving::dropLoot to see exactly everything need to do.

**Comment by mxnmnm** [comment]
I could have sworn he said he wouldn't fix it but I guess I'm thinking of something else maybe lol

**Comment by largeorbitalobject** [comment]
Hello, was there ever anything done to fix this? Not only does this work with zombies but also donkeys and other mobs that can hold your items.

**Comment by mxnmnm** [comment]
Don't think so, I'd just disable it if I were you.
